source/enclave.haml

```diff
@@ -40,71 +52,90 @@
         run your ops, allowing you to focus on development.
 
     .feature-carousel__illustration.grid-item
-      = partial '/images/enclave-features/stack.svg'
+      %img{ src: '/images/enclave-features/stack.png' }
 
   .feature-carousel.grid-container.grid--2up.grid--2up-flex.feature-carousel--network
     .feature-carousel__details.grid-item
       .heading__subtitle The Network
-      %h2.heading__title= lorem.words 4
-      %p.heading__summary= lorem.paragraph
+      %h2.heading__title.heading__title--dark= lorem.words 4
+      %p.heading__summary.heading__summary--light-blue-gray= lorem.paragraph
```

Comment:
@fancyremarker - these are the `lorem` bits needing content for the enclave feature carousel